homepage.md

```diff
@@ -32,6 +32,9 @@ Statistics about stored and transferred data:
 - **storage_total_pieces** - total number of pieces stored on storage nodes
 - **storage_total_segments** - total number of segments stored on storage nodes
 - **storage_free_capacity_estimate_bytes** - statistical estimate of free storage node capacity, with suspicious values removed
+- **storage_after_expansion_total_bytes** - total number of bytes stored on the network after expansion
```

Comment:
I noticed this is not a whole number, e.g. `1493873144708125.8`. Is this a bug?

Reply:
Already fixed this morning.  It was caused by our [BI] datasources not being updated properly. If you check again you should see the right values.